### PR TITLE
adding nscd cookbook parameter

### PR DESCRIPTION
**Pull request by acaiafa**
### Description

adding default['nscd']['cookbook'] so incase people need to override the default template or anything else this will give them the ability to do so. 

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ x] All tests pass. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] The CLA has been signed. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD

### CODE REVIEW

**Comment by tas50** [comment]
The attribute name seems a bit confusing. It might make more sense as template_cookbook. Also add it to the readme. Otherwise the idea is solid and I can get it merged in and released.

**Comment by acaiafa** [comment]
Ok @tas50 I updated with template_cookbook. I think thats a great idea. Thanks!

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Installs and configures name service cache daemon (nscd). Provides a custom reso
 - compat_resource
 
 ## Attributes
-
+- `default['nscd']['template_cookbook']` - nscd cookbook name. You can ovverride this attribute if you want to pass in a custom template for nscd.conf
```
Comment (glensc) [inline README.md]: ovverride spelling error

File: attributes/default.rb [hunk 2]
```diff
@@ -19,6 +19,9 @@
 
 require 'etc'
 
+# nscd cookbook(incase you need to override anything in a wrapper)
```
Comment (glensc) [inline attributes/default.rb]: "incase" is two words: "in case"

**Comment by acaiafa** [comment]
@glensc resolved.

**Pull request merged**